doc/ref_model/chapters/chapter04.md

```diff
@@ -282,43 +282,75 @@ The idea of the Cloud Infrastructure profiles is to have a predefined set of inf
-| Memory                                | Virtual compute instance memory in megabytes.                                                                                                                                                           |
+| Compute Flavour Name                  | A descriptive name |
+| Compute Flavour ID                  | Flavor unique identifier |
+| Number of vCPUs | Number of virtual compute resources (vCPUs) presented to the VM instance.  |
+| Memory                                | Virtual compute instance memory in megabytes.  |
 | Ephemeral/Local Disk                  | Specifies the size of an ephemeral data disk that exists only for the life of the instance. Default value is 0.<br />The ephemeral disk may be partitioned into boot (base image) and swap space disks. |
-| Management Interface                  | Specifies the bandwidth of management interface/s                                                                                                                                                       |
-
+| Root Disk                  | Specifies the size of '/' root partition. It can be considered an Ephemeral Disk |
+| CPU Parameters                  | Capability to associate virtual CPUs to the physical CPUs, and enable specific CPU architectures |
+| NUMA                  | Allow system memory allocation into cells or nodes that are associated with particular CPUs (there could be performance degradation due to an high number of requests through this shared bus) |
+| Network Parameters                  | Enable network capabilities allowing improvements on network performance (make sure that current deployment allows it, otherwise, it can lead either to a decrease on performance or instability) |
 <p align="center"><b>Table 4-12:</b> Compute Flavour Geometry Specification.</p>
 
 <a name="4.2.1.1"></a>
-#### 4.2.1.1 Predefined Compute Flavours
-The intent of the following Flavours list is to be comprehensive and yet effective to cover both IT and NFV workloads. The compute Flavours are specified relative to the “large” Flavour. The “large” Flavour configuration consists of 4 vCPUs, 8 GB of RAM and 80 GB of local disk, and the resulting virtual compute instance will have a management interface of 1 Gbps. The “medium” Flavour is half the size of a large and small is half the size of medium. The tiny Flavour is a special sized Flavour.
+#### 4.2.1.1 Predefined Compute Flavors
+
+The following tables provide a descriptive approach on how a Compute flavor can be created to suit the needs of an application, so it can be onboarded. This will minimize several challenges related to Infrastructure management:
+
+* Applications can be categorized based on their resource requirements towards cloud infrastructure
+* Additional flavors can be created which are linked to  to different VM requirements
```

Comment:
Aren't Parameterised Compute Flavours the mechanism when predefined Compute Flavours do not meet the needs of a VM request?

Reply:
@pgoyal01 Let me propose a different statement